### PR TITLE
8123456: remove test from exclude list

### PR DESCRIPTION
**Pull request by AlanBateman**
/reviewer add chegar
/reviewer add michaelm

---------
### Progress
- [x] Change must not contain extraneous whitespace

### Issue
 * [JDK-8123456](https://bugs.openjdk.java.net/browse/JDK-8123456): Unable to select last word ⚠️ Title mismatch between PR and JBS.

### Reviewers
 * Michael McMahon ([michaelm](@Michael-Mc-Mahon) - **Reviewer**) ⚠️ Review applies to da1c3b8ac27f6be7b8bf1b726556df9d768bba00

### Download
`$ git fetch https://git.openjdk.java.net/playground pull/30/head:pull/30`
`$ git checkout pull/30`

### CODE REVIEW

File: test/jdk/ProblemList.txt [hunk 1]
```diff
@@ -633,8 +633,6 @@ java/nio/channels/DatagramChannel/Unref.java                    8233519 generic-
 
 java/nio/channels/AsynchronousSocketChannel/StressLoopback.java 8211851 aix-ppc64
 
-java/nio/channels/Selector/Wakeup.java                          6963118 windows-all
-
 ############################################################################
```
Comment (Michael-Mc-Mahon) [inline test/jdk/ProblemList.txt:636]: Looks dodgy to me!

**Review by Michael-Mc-Mahon — CHANGES_REQUESTED** [review]
Remove line 634 also

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (AlanBateman) [reply test/jdk/ProblemList.txt:636]: how so?

Comment (ChrisHegarty) [reply test/jdk/ProblemList.txt:636]: looks doggy to me too! ;-)

**Review by ChrisHegarty — CHANGES_REQUESTED** [review]

File: test/jdk/ProblemList.txt (hunk 1 above)
Comment (Michael-Mc-Mahon) [reply test/jdk/ProblemList.txt:638]: It's okay actually, see other comment

**Review by Michael-Mc-Mahon — APPROVED** [review]
It's fine now

**Pull request closed without merging**